recipes/dorado/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dorado" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://cdn.oxfordnanoportal.com/software/analysis/dorado-{{ version }}-linux-x64.tar.gz  # [linux]
```

Comment:
I just downloaded version 0.7.1 to get the SHA256 - it downloaded 2.4GB file! This is quite enterpricey !
0.6.0 is 2.0 GB
Is this OK ?